cmd/mt-index-migrate-050-to-054/main.go

```diff
@@ -50,18 +50,18 @@ func main() {
 		log.Fatalf("failed to create cql session. %s", err)
 	}
 	wg.Add(1)
-	go writeDefs(session, defsChan, wg)
+	go writeDefs(session, defsChan, &wg)
```

Comment:
In other tools, i have just made the waitGroup a package variable rather than passing it to the goroutines.